larray/core/session.py

```diff
@@ -873,8 +946,9 @@ def elem_equal(e1, e2):
 
     array_equals = renamed_to(element_equals, 'array_equals')
 
+    # TODO: include metadata?
```

Comment:
This could be an option. But we need a default value and I am unsure about it. Do you have an idea?

Reply:
IMO metadata should not be tested in equals. There are not properly items of a Session.

Reply:
ok, let's go for this. Should be documented explicitly though.

Reply:
The docstring starts with "*Test if groups, axes and arrays of the current session are equal to those of another session*". It's quite explicit no?

Reply:
Not enough I think. Users have to think for themselves that "hey metadata is not in that list". I think there should be a note somewhere stating something vaguely like: "other metadata (from .meta) is **NOT** compared" or "is ignored".